docs/getting-started/setup.md

```diff
@@ -45,6 +46,14 @@ While syntax highlighting of `.gradle.dcl` files works out of the box in Studio
 2. Enable the `gradle.declarative.studio.support` and `gradle.declarative.ide.support` flags
 3. Restart the IDE
 
+### IntelliJ IDEA
+
+Download and install a special IntelliJ IDEA Nightly release.
+You can find the promoted nightly releases in [this Google Drive folder](https://drive.google.com/drive/folders/19iWu2F4dYs0Vc9xFMkSwY7wizKk-YMKl19C5EMRgENKU_tOAwOVLcIGZ6Bbm63Q7V) for macOS (Apple Silicon and Intel), Windows and Linux.
```

Comment:
This link is broken, should be https://drive.google.com/drive/folders/19iWu2F4dYs0Vc9xFMkSwY7wizKk-YMKl

Reply:
Ah thanks! Wrong copy/paste. Fixed